sites/dentistryiq.com/config/gcse.js

```diff
@@ -1,5 +1,3 @@
-const GCSEConfiguration = require('@base-cms/marko-web-gcse/config');
+const configureGCSE = require('@endeavor-business-media/package-shared/config/gcse');
 
-const config = new GCSEConfiguration('003355913687346718228:prps-rbhnq0');
-
-module.exports = config;
+module.exports = configureGCSE({ apiKey: '003355913687346718228:266mmrxk-zy' });
```

Comment:
The `apiKey` is different. Is this intentional?

Reply:
It was not. I'll get that corrected.